src/main/java/com/lohika/morning/ecs/domain/settings/SettingsService.java

```diff
@@ -0,0 +1,31 @@
+package com.lohika.morning.ecs.domain.settings;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class SettingsService {
+  private final SettingsRepository settingsRepository;
+
+  public Settings getSettings() {
+    List<Settings> all = settingsRepository.findAll();
+
+    if (all.size() > 1) {
+      throw new RuntimeException("Something going wrong. Can not get settings.");
```

Comment:
Let the message be explicit: "At most 1 settings record should be in DB, but {N} found"